tools/python/calc_func_checksums.py

```diff
@@ -12,7 +12,7 @@
 
 FUNCTIONS_TO_CALC = [
     # function         checksum variable        func size variable
-    ('func_80019808', 'gFunc80019808Checksum', 'gFunc80019808Length'),
+    ('race_check_finish', 'gFunc80019808Checksum', 'gFunc80019808Length'),
     ('render_scene', 'gTractionTableChecksum', 'gTrackRenderFuncLength'),
     ('obj_loop_goldenballoon', 'gObjLoopGoldenBalloonChecksum', 'gObjLoopGoldenBalloonLength'),

```

Comment:
Should the checksum variable not be called `gRenderSceneChecksum` and the func size variable `gRenderSceneFuncLength`? 

```suggestion
    ('obj_loop_goldenballoon', 'gRenderSceneChecksum', 'gRenderSceneFuncLength'),
```